packaging/freebsd/README.md

```diff
@@ -0,0 +1,22 @@
+mackerel-agent for FreeBSD
+==========================
+
+1. Get binary for FreeBSD at release page. https://github.com/mackerelio/mackerel-agent/releases
+    - `amd64`: `mackerel-agent_freebsd_amd64.tar.gz`
+    - `i386` : `mackerel-agent_freebsd_386.tar.gz`
+    - (`arm` : `mackerel-agent_freebsd_arm.tar.gz`)
+2. Extract file, and `cd`.
+    - `tar -xzvf mackerel-agent_freebsd_*.tar.gz`
+3. Copy `mackerel-agent` to `/usr/local/bin/`.
+    - `(sudo) cp mackerel-agent /usr/local/bin/`
+4. Edit `mackerel-agent.conf`, after copy to `/usr/local/etc/`.
```

Comment:
[nit]
```suggestion
4. Edit `mackerel-agent.conf`, then copy it to `/usr/local/etc/`.
```

Reply:
thanks, I fix it at https://github.com/mackerelio/mackerel-agent/pull/559/commits/67900a9bf9fd4f584decba95d704816af7c486f9 .